docs/MANUAL.md

```diff
@@ -0,0 +1,215 @@
+
+```ini
+# Code within this type and all descendants is considered test code.
+# Defaults to DUnit's test base type: TestFramework.TTestCase.
+sonar.delphi.testType=TestFramework.TTestCase
+# Code within any type annotated with this attribute is considered test code.
+# Defaults to DUnitX's test attribute: DUnitX.Attributes.TestFixtureAttribute.
+sonar.delphi.testAttribute=DUnitX.Attributes.TestFixtureAttribute
+```
+
+For the full list of SonarDelphi configuration options, see [here](CONFIGURATION.md).
+
+## Run a SonarDelphi scan
+
+Once your project is configured, you simply
+[run the scanner](https://docs.sonarsource.com/sonarqube/latest/analyzing-source-code/scanners/sonarscanner/)
+in the same directory as your `sonar-project.properties`.
+
+After an analysis, the project page on SonarQube will refresh with updated issues and metrics.
+Browsing the issues, you have the ability to update them if needed - assign them to other users, add comments,
+or change their status (e.g. accepting issues you won't fix or hiding false positives).
+
+If there are a large number of issues for rules that don't suit your codebase - for example, issues complaining
+your classes don't begin with `T` when you have a different convention - you may benefit from
+[customizing SonarDelphi's analysis rules](#configure-analysis-rules).
+
+## Configure analysis rules
+
+The rules applied during a scan are dictated by the project's
+[quality profile](https://docs.sonarsource.com/sonarqube/latest/instance-administration/quality-profiles/).
+
+For Delphi, the default quality profile ("Sonar way") contains sensible default rules that are generally
+accepted to be best practice. These may not be helpful to apply to your codebase if it has its own conventions that
+conflict with the rules. Some common quality profile changes include:
+
+* Setting `excludeApi` to `true` (allowing "unused" public API) on `UnusedConstant`, `UnusedField`,
+  `UnusedGlobalVariable`, `UnusedProperty`, `UnusedRoutine`, and `UnusedType`
+* Changing the `prefixes` field on `ClassName`, `RecordName`, `UnitName`, `EnumName`, `InterfaceName`,
+  and `HelperName`
+* Disabling `ConstantName`
+* Disabling `ShortIdentifier`
+* Disabling `NilComparison`
+
+While SonarDelphi's quality profile contains many useful rules, there are also many non-default rules
+that are more niche or opinionated. Some rules that are often added to quality profiles include:
+
+* `BeginEndRequired` ('begin'..'end' should always be used)
+* `WithStatement` ('with' statements should not be used)
+* `GroupedFieldDeclaration` (Fields should be declared individually)
+* `InterfaceGuid` (Interfaces should have unique GUIDs)
+* `CatchingRawException` (Raw exceptions should not be caught)
+* `DateFormatSettings` (Date formatting should not use the default TFormatSettings)
+
+Using quality profiles allows you to track the quality of your code as *you* define it.
+
+### Customize analysis using template rules
```

Comment:
This could be organized more like:

### Custom rules

#### Template rules

...

#### Custom rules plugin

...